docs/usage/5-responses/4-response-headers.md

```diff
@@ -228,3 +228,56 @@ router = Router(
 
 # ...
 ```
+
+## Specific Headers Implementation
+
+Starlite has a dedicated implementation for a few headers that are commonly used. These headers can be set separately
+from `response_headers` in dedicated kwargs available on all layers of the app (individual route handlers, controllers,
+routers and the app itself).
+
+### Cache Control
+
+Starlite has a dedicated [CacheControlHeader][starlite.datastructures.CacheControlHeader] class for
+[`Cache-Control`](https://developer.mozilla.org/en-US/docs/Web/HTTP/Headers/Cache-Control) which allows easy creation
+and parsing of the header value and provides accessors to its directives.
+
+Here is a simple example that shows how to use it:
+
+```python
+import time
+
+from starlite import get, Router, Controller
+from starlite.datastructures import CacheControlHeader
+
+
+class MyController(Controller):
+    cache_control = CacheControlHeader(max_age=86_400, public=True)
+
+    @get("/chance_of_rain")
+    def get_chance_of_rain(self) -> float:
+        return 0.5
+
+    @get("/timestamp", cache_control=CacheControlHeader(no_store=True))
+    def get_server_time(self) -> float:
+        return time.time()
+
+
+@get("/population")
+def get_population_count() -> int:
+    return 100000
+
+
+router = Router(
+    route_handlers=[MyController, get_population_count],
+    cache_control=CacheControlHeader(max_age=2_628_288, public=True),
+)
```

Comment:
As this looks like a complete example, please have a look at #622 for how we prefer to include complete examples in the docs.

You can write the example as a script, include a test for it, and then cross reference the script in the docs.

The example tests are run in CI so this ensures that our documentation examples don't go stale.

Reply:
Thanks! Fixed here: https://github.com/starlite-api/starlite/pull/601/commits/7dc701b993ca6f097558cf2df6f32584ae6128da